src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="cal">
+      <div class="cal__day"></div>
+      <div class="cal__day"></div>
+      <div class="cal__day"></div>
+      <div class="cal__day"></div>
+      <div class="cal__day"></div>
+      <div class="cal__day"></div>
+      <div class="cal__day"></div>
+      <div class="cal__day"></div>
+      <div class="cal__day"></div>
+      <div class="cal__day"></div>
+      <div class="cal__day"></div>
```

Comment:
no need to shorten words, it's a good practice to give your elements valuable readable classes, better use 'calendar' and 'calendar__day'